### PR TITLE
A simple script to get latest package version from PyPI

### PR DESCRIPTION
**Pull request by ilevkivskyi**
This may be useful together with https://github.com/python/typeshed/pull/4291 and https://github.com/python/typeshed/pull/4259

### CODE REVIEW

**Comment by ilevkivskyi** [comment]
> Looks good! Maybe we should put the new scripts to some subdirectory?

OK, I will update all three PRs to move them under `/scripts`.

**Comment by Akuli** [comment]
Is there a naming convension for the `scripts` directory? `migrate_script.py` ends with `_script`, for example.

If there is `scripts/foo.py`, does that make mypy or other type checkers think that there exists a module named `foo` that you can import from any file?

**Comment by JelleZijlstra** [comment]
@Akuli these scripts are purely for internal use in the repo; type checkers should not pick them up as actual stubs.

File: scripts/get_version.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    )
+    with requests.Session() as session:
+        session.mount("https://", HTTPAdapter(max_retries=retry_strategy))
+        resp = session.get(url, timeout=TIMEOUT)
+    if not resp.ok:
+        raise ValueError("Error while retrieving version")
+    data = resp.json()
+    if not version:
+        # Use the METADATA.toml version, if not given one.
+        version = read_base_version(distribution)
```
Comment (hauntsaninja) [inline scripts/get_version.py:44]: `max("2.1.3", "2.1.10") == "2.1.3"`, so we should do:
```
releases = [tuple(map(int, release.split("."))) for release in data["releases"]]
```
etc.

I'd also add an `assert version.count(".") == 1`

**Review by hauntsaninja — CHANGES_REQUESTED** [review]

**Comment by ilevkivskyi** [comment]
This will be added to a repository with more restricted write access (for security reasons).

File: scripts/get_version.py (hunk 1 above)
Comment (ilevkivskyi) [reply scripts/get_version.py:44]: Thanks! I added a TODO about this (in a different repo).

**Pull request closed without merging**